contents/blog/best-open-source-ab-testing-tools.md

```diff
@@ -0,0 +1,165 @@
+
+<ArrayCTA />
+
+**Further reading:**
+- [The best open source Hotjar and FullStory alternatives](/blog/best-open-source-session-replay-tools)
+- [What we've learned about running effective A/B tests](/blog/experiments)
+- [The differences between PostHog and Amplitude](/blog/posthog-vs-amplitude)
+
+## 2. [GrowthBook](https://www.growthbook.io/)
+
+![GrowthBook - best open source ab testing tools](../images/blog/open-source-testing-tools/growthbook.png)
+
+GrowthBook is an open-source platform for feature flags and A/B tests which helps teams to deploy code efficiently and analyze experiments. Built by engineers who wanted better adoption insights into their releases, it is a modular solution that promotes feature flagging as an essential step in the development process and can be used as a full-stack platform, a plugin feature flagging tool or an analysis engine.
+
+### Who is GrowthBook for?
+
+GrowthBook is built for data, engineering and product teams who need the power of a customizable platform without having to build it in house.
+
+### Strengths
+
+- Modular platform to adapt to your needs
+- Lightweight SDKs for speed
+- Easy to implement – no need for data or engineering resources
+- Self-hosted and managed options available
+
+### How much does it cost?
+
+GrowthBook open source is free and includes unlimited experiments. A hosted version includes a visual A/B test editor and is free up to 5 users, with pricing at $20 per user per month after.
+
+### Open sources license
+
+GrowthBook is available under an MIT license. Visit the [Growthbook GitHub repo](https://github.com/growthbook/growthbook/) for more info. 
+
+## 3. [Unleash](https://www.getunleash.io/)
+
+![Unleash - best open source ab testing tools](../images/blog/open-source-testing-tools/img2_Unleash.png)
+
+Unleash is a feature management platform which provides an overview of all features across your applications and services. The platform empowers engineering teams to implement A/B tests via feature toggles and offers accurate user targeting.
+
+You can use Unleash to define the rules of your experiments, however the platform doesn’t provide all the tools you need to manage A/B tests end-to-end and you will need to connect your experiment with an analytics platform like Google Analytics or Mixpanel in order to visualize results.
+
+### Who is Unleash for?
+
+Unleash is for autonomous development teams who want to be free from constraints and get on with testing and validating new features safely. They build with key developer concerns in mind: integration, privacy, resilience and performance.
```

Comment:
'who want to be free from constraints' sounds really non-specific. This section almost reads like it's been copied and pasted from Unleash's marketing materials.